packages/odyssey-storybook/src/contributing/CustomTheme.mdx

```diff
@@ -27,3 +27,17 @@ Visually, the mui components under `OdysseyThemeProvider` with custom theme woul
 <Canvas>
   <Story id="customization-components--radio-group-story" />
 </Canvas>
+
+### Customizming MUI Components
```

Comment:
this is just an example to show the customization of mui components. I'll remove this if we don't want this to be in the storybook.

Reply:
I like the live example!